tests/integration/test_integ.py

```diff
@@ -0,0 +1,129 @@
+# HOW TO EXECUTE?
+# First install the package: pip install pytest-docker-compose
+# Run with: pytest --docker-compose="path_to_docker_compose_yml" test_integ.py
+
+import pytest
+import os
+import subprocess
+import unittest
+import time
+import sys
+import logging
+
+
+class TestIntegratedTool(unittest.TestCase):
+
+
+    def setUp(self):
```

Comment:
with `setUp` you would check for all that container running stuff before every test method, `setUpClass` should be the way to go

Reply:
done